### PR TITLE
docs(angularjs): instruction to deploy this framework

### LINKED ISSUE #1179: Document how to deploy an AngularJS application
This page is part of the answer: https://doc.scalingo.com/languages/nodejs/start#framework-requiring-to-serve-static-files

But a step by step tutorial would be nice.

https://app.intercom.com/a/apps/w4oogu7s/inbox/inbox/conversation/12375700021544#part_id=comment-12375700021544-8106237632
https://app.intercom.com/a/inbox/w4oogu7s/inbox/shared/all/conversation/12375700146387?view=List

### PR DESCRIPTION
**Pull request by EtienneM**
Fix #1179

### CODE REVIEW

File: src/_posts/platform/app/2000-01-01-static-files-hosting.md [hunk 1]
```diff
@@ -46,12 +48,36 @@ Don't forget to add a `package.json` file in order to add the `express` dependen
...
+
+You also need to add a Procfile to instruct Scalingo how to boot Nginx. The Procfile content should be:
+
+```yaml
+web: bin/run
+```
+
+Push your code to Scalingo, and the application is up and running, ready to serve your static files.
+
 ## Using Jekyll
```
[the remark this answers is not in the document]
Comment (yohann-bacha) [reply src/_posts/platform/app/2000-01-01-static-files-hosting.md:75]: I am quite okay with this idea, are there any other static sites generators documented?

In any case, if we make a page about it, we should rewrite the section

File: src/_posts/languages/nodejs/2000-01-01-angularjs.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+title: Deploying AngularJS on Scalingo
+nav: AngularJS
+modified_at: 2023-02-16 00:00:00
+tags: nodejs angularjs
+index: 4
+---
+
+AngularJS applications can be hosted on Scalingo. It requires some specific instructions as usually the application generate static files during the deployment. And these files requires a web server to be served to your clients.
```
Comment (yohann-bacha) [inline src/_posts/languages/nodejs/2000-01-01-angularjs.md:9]:
```suggestion
AngularJS applications can be hosted on Scalingo. It requires some specific instructions as usually the application generates static files during the deployment. And these files require a web server to be served to your clients.
```

File: src/_posts/languages/nodejs/2000-01-01-angularjs.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+```nginx
+root /app/dist;
+
+location / {
+  try_files $uri $uri/ /index.html =404;
+}
+```
+
+Then instruct Scalingo how to start the application with a [`Procfile`]({% post_url platform/app/2000-01-01-procfile %}):
```
Comment (yohann-bacha) [inline src/_posts/languages/nodejs/2000-01-01-angularjs.md:30]: "Instruct" doesn't seem that smooth to me 
```suggestion
Then tell Scalingo how to start the application with a [`Procfile`]({% post_url platform/app/2000-01-01-procfile %}):
```

File: src/_posts/platform/app/2000-01-01-static-files-hosting.md [hunk 4]
```diff
@@ -46,12 +48,36 @@ Don't forget to add a `package.json` file in order to add the `express` dependen
...
+root /app/dist;
+
+location / {
+  try_files $uri $uri/ /index.html =404;
+}
+```
+
+This configuration assumes that your static files are located in the `dist` folder of your application.
+
+You also need to add a Procfile to instruct Scalingo how to boot Nginx. The Procfile content should be:
```
Comment (yohann-bacha) [inline src/_posts/platform/app/2000-01-01-static-files-hosting.md:67]:
```suggestion
You also need to add a `Procfile` to instruct Scalingo how to boot Nginx. The `Procfile` content should be:
```

**Review by yohann-bacha — CHANGES_REQUESTED** [review]

File: src/_posts/languages/nodejs/2000-01-01-angularjs.md (hunk 3 above)
Comment (EtienneM) [reply src/_posts/languages/nodejs/2000-01-01-angularjs.md:30]: "tell": de ce que je sais, ce mot signifie plutôt "raconter" dans le sens "raconter des histoires".
"instruct" (https://www.wordreference.com/enfr/instruct) serait dans le sens "donner l'ordre"

**Review by yohann-bacha — APPROVED** [review]

**Pull request merged**